continuous-deployment.md

```diff
@@ -0,0 +1,20 @@
+Continuous Deployment
+-----
+
+Continuous Deployment (CD) is a software release process where verified changes to a codebase are deployed immediately and autonomously to a production environment.
+
+You should have continuous deployment set up for your repository, with the following behaviours configured if applicable:
+* Automatically deploying changes to the `main` branch to production
+* Using an AMIgo AMI and regularly redeploying the application on a schedule
+* Restricting deployments if and when required
+
+You should already have [continuous integration](continuous-integration.md) configured in order to set up continuous deployment.
```

Comment:
A small Markdown note to say you can use the link text itself, and declare the link it should point to further down. It often reads nicer.

```suggestion
You should already have [continuous integration] configured in order to set up continuous deployment.

[continuous integration]: continuous-integration.md
```

Reply:
Oh, nice! Didn't know this is a thing, thanks for sharing it.